wp-content/themes/csisnuclearnetwork/inc/site-nav-walker.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Custom walker for main navigation
+ *
+ * @package Nuclear Network
+ */
+
+class NuclearNetwork_Menu extends Walker_Nav_Menu {
+
+    function start_lvl( &$output, $depth = 0, $args = array() ) {
+				// Only show the featured content if we have a URL.
+        if ( $args->has_featured === true && $args->featured_post ) {
+          $featured_post = $args->featured_post[0];
+          $featured_post_id = $featured_post->ID;
+            $title = mb_strimwidth(get_the_title($featured_post_id), 0, 59, '...');
+            $permalink = get_permalink($featured_post_id);
+            $thumbnail = get_the_post_thumbnail_url($featured_post_id);
+            $post_type = get_post_type($featured_post_id);
+
+            if (function_exists('coauthors_posts_links')) {
+              $authors = '';
+        
+              foreach (get_coauthors($featured_post_id) as $coauthor) {
+                $name = $coauthor->display_name;
+        
+                $authors .= '<div class="site-nav__feat-post-author">' . $name . '</div>';
+              }
+            } else {
+              $authors .= the_author_posts_link($featured_post_id);
+            }
+
+            $author_list = '<div class="site-nav__feat-post-authors text--bold">' . $authors . '</div>';
+            
+            $featured_post_html = '<div class="site-nav__feat-post-terms-type text--bold">Featured</div>
+            <a href="' . esc_url ( $permalink ) . '" class="site-nav__feat-post-title text--bold">' . $title . '</a>' .
+            $author_list . '
+            <div class="site-nav__feat-post-thumbnail"><a href="' . esc_url ( $permalink ) . '"> <img src="' . esc_url ( $thumbnail ) . '"></a></div>';
+          }
+
+        $class_names = $value = '';
+        $classes = empty( $item->classes ) ? array() : (array) $item->classes;
+        $classes[] = 'sub-menu sub-menu-depth-' . $depth;
+        $class_names = join( ' ', apply_filters( 'nav_menu_css_class', array_filter( $classes ), $item, $args ) );
+        $class_names = $class_names ? ' class="' . esc_attr( $class_names ) . '"' : '';
+
+        $indent = str_repeat("\t", $depth);
+        $output .= "\n$indent<div class='sub-menu-container sub-menu-container-depth-" . $depth . " row'>
+            <div class='col-xs-12 col-md sub-menu-children' aria-expanded='false'>\n";
```

Comment:
Whenever you see classes like `col-xs-12` or `col-md`, you can remove those. The ASP site uses a Bootstrap derivative stylesheet based on flexbox that has a 12 column layout and that's what those classes are from. NN doesn't use that, so go ahead and remove it.